styles/components/showfps.scss

```diff
@@ -0,0 +1,34 @@
+@use '../config' as *;
+@use '../abstract/mixin';
+
+.fpsmeter {
+	width: 100%;
```

Comment:
Does this need to be `width: 100%`/`height: 100%`? Even if it's top-level can't we just position without that?

Reply:
We can yeah, if we ever want other panels below it. I'd like it to be absolutely positioned in a place that makes more sense for UI in the future

Reply:
Yeah I think this is fine until we do more absolute positioning (HUD customization)